client/blocks/login/index.jsx

```diff
@@ -555,6 +558,7 @@ export default connect(
 				redirectTo: stateProps.redirectTo,
 				loginFormFlow: true,
 				showGlobalNotices: true,
+				flow: ownProps.isJetpack ? 'jetpack' : null,
```

Comment:
This allows us to tell WPCOM's endpoint that the user is going through a Jetpack flow.